modules/ROOT/pages/gs-adding-custom-3scale-routes.adoc

```diff
@@ -13,8 +13,8 @@ For example, you might want to expose an API as `https://mydomain.com/v1/getStuf
 To use a custom domain, you must create a route in the OpenShift 3scale project as described below.
 
 .Prerequisites
-* the `customer-admin` user credentials
-* a custom domain registration
+* The `customer-admin` user credentials
```

Comment:
what's the thinking here? these are not sentences

Reply:
I thought, for consistency. The other prereqs are capitalized, but I guess it's not crucial.